apps/protocol-api/src/prisma/resolvers/Contribution.ts

```diff
@@ -696,4 +712,33 @@ export class ContributionCustomResolver {
       }
     });
   }
+
+  @TypeGraphQL.Query(_returns => [ContributionCountByUser], {
+    nullable: false,
+  })
+  async getDaoContributionCountByUser(
+    @TypeGraphQL.Ctx() { prisma }: Context,
+    @TypeGraphQL.Args() args: GetContributionArgs,
+  ) {
+    const start = args.where.startDate;
+    const end = args.where.endDate;
+    const guildId = args.where.guildId;
+
+    const result = await prisma.$queryRaw<ContributionCountByUser>`
+      SELECT  count(gc.id) as count,
+              u.id as "user_id",
+              u.display_name as "user_name"
```

Comment:
Wondering if we may want to standardize when/how we use `display_namename` instead of `name` -- our profile call to `user.create` sets the username (which I think is the `name` field in the user table) 

Does Kevin Malone write to `display_name`? Curious mainly so we're on the same page -- this looks great, that's my only question!

Reply:
Kevin updates both name and display_name in the User table. but I agree with you that it's not immediately clear why there are two separate fields for this. @alexkeating do you have an opinion on this?

Reply:
Yes, we should probably consolidate. I think originally display name was meant to be discord name and the name was like there user name but I think we added display name to the discord table making the distinction irrelevant in user